apps/meteor/client/hooks/useContinuousSoundNotification.ts

```diff
@@ -0,0 +1,53 @@
+import type { ICustomSound } from '@rocket.chat/core-typings';
+import { useSetting, useUserPreference, useUserSubscriptions } from '@rocket.chat/ui-contexts';
+import { useEffect } from 'react';
+
+import { CustomSounds } from '../../app/custom-sounds/client/lib/CustomSounds';
+
+const query = { t: 'l', ls: { $exists: false }, open: true };
+export const useContinuousSoundNotification = () => {
+	const userSubscriptions = useUserSubscriptions(query);
+
+	const playNewRoomSoundContinuously = useSetting('Livechat_continuous_sound_notification_new_livechat_room');
+
+	const newRoomNotification = useUserPreference<string>('newRoomNotification') || '';
+	const audioVolume = useUserPreference<number>('notificationsSoundVolume');
+
+	const continuousCustomSoundId = `${newRoomNotification}-continuous`;
```

Comment:
What happens if the newRoomNotification has its default value (`''`)? Could this `-continuous` id be a problem?